ecs.php

```diff
@@ -395,4 +396,14 @@
     $services->set(FinalClassInEntitiesFixer::class);
 
     $services->set(AboveTwoArgumentsMultilineFixer::class);
+
+    $services
+        ->set(HeaderCommentFixer::class)
+        ->call('configure', [[
+            "header" => "This file has been created by developers from BitBag. \nFeel free to contact us once you face any issues or want to start\nYou can find more information about us on https://bitbag.io and write us\nan email on hello@bitbag.io.",
```

Comment:
I would put the content into constant.
Additionally, I would make environment variable to disable this check due to no need for OS header in commercial and private projects.

Reply:
We decided to not put header content in the constant. Changes regarding adding environment variable are done in next commit.